presto-main/src/test/java/io/prestosql/sql/planner/TestLocalDynamicFilter.java

```diff
@@ -128,17 +132,18 @@ public void testNone()
         LocalDynamicFilter filter = new LocalDynamicFilter(
                 ImmutableMultimap.of("123", new Symbol("a")),
                 ImmutableMap.of("123", 0),
+                TypeProvider.copyOf(ImmutableMap.of(new Symbol("a"), INTEGER)),
                 1);
         assertEquals(filter.getBuildChannels(), ImmutableMap.of("123", 0));
         Consumer<TupleDomain<String>> consumer = filter.getTupleDomainConsumer();
-        ListenableFuture<TupleDomain<Symbol>> result = filter.getResultFuture();
+        ListenableFuture<Map<Symbol, Domain>> result = filter.getResultFuture();
 
         assertFalse(result.isDone());
         consumer.accept(TupleDomain.withColumnDomains(ImmutableMap.of(
```

Comment:
change it to `consumer.accept(TupleDomain.none())`

Reply:
done